repoindex.md

```diff
@@ -261,6 +261,34 @@ organization in GitHub under
 		</tr>
 	</table>
 </fieldset>
+<fieldset class="repo">
+	<legend>
+		pmem-related workloads
+	</legend>
+	<p>
+	These repos contain non-volatile memory related workloads.
+	Their concept is based on combined market ready solutions with
+	persistent memory. Workloads are delivered in SaaC (Solution as
```

Comment:
Is the acronym "SaaC" a thing or our own invention? I googled a bit and couldn't find anything relevant.
If it's our own invention then I'd drop it.

Reply:
dropped